src/pages/CreditCardRegister/components/CreditCardCVCInput.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import Input from '../../../components/Input';
+import * as S from '../style';
+
+type Props = {
+  creditCardCVC: string,
+  setCreditCardCVC: React.Dispatch<React.SetStateAction<string>>,
+};
+
+function CreditCardCVCInput({ creditCardCVC, setCreditCardCVC }: Props) {
+  const [error, setError] = useState(false);
+
+  const handleChangeCreditCardCVC = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newCVC = event.target.value.replace(/\D/g, '');
+    if (newCVC.length <= 3) {
+      setCreditCardCVC(newCVC);
+    }
+  };
+
+  useEffect(() => {
+    if (creditCardCVC.length > 0 && creditCardCVC.length < 3) {
+      setError(true);
+    } else {
+      setError(false);
+    }
+  }, [creditCardCVC]);
```

Comment:
일단 코드를 리팩토링 해보자면!
```suggestion
  useEffect(() => {
    const isError =creditCardCVC.length > 0 && creditCardCVC.length < 3;
    setError(isError);
  }, [creditCardCVC]);
```

이렇게 수정해보시면 좋을 것 같고, 애초에 useEffect 가 없어도 될 것 같은데 아래와 같이 변수로 사용하시는건 어떤가요?

```suggestion
  const isError = creditCardCVC.length > 0 && creditCardCVC.length < 3;
```

Reply:
코드가 훨씬 간결해 진 것 같습니다..!